web/styles/app_components.css

```diff
@@ -1158,6 +1158,7 @@ input.settings_text_input {
     justify-content: space-between;
     outline: none;
     color: var(--color-text-default);
+    width: 206px;
```

Comment:
Should we document this value and/or put it into a CSS variable? Minor nit. But it remaining as a hard-coded pixel value is different from other changes in this PR.

Reply:
That's fair, though it's hard for me to add documentation for this because I don't have much context on it. This line comes from a commit that moves this style from a modal specific stylesheet to this more general stylesheet, and I feel okay keeping it the way it was there (since it's sort of net neutral and I'm not using this value beyond that).

Reply:
I guess it wouldn't be that hard to make a variable like `default-input-width` or something? But I don't think it's necessary for this PR and could be a followup.

Reply:
Sure, if you're willing to track it.

Reply:
Yeah sure!